### PR TITLE
task_DEV-1

### PR DESCRIPTION
**Pull request by TatianaGantievskaya**

### CODE REVIEW

File: TheLongestSubsequence/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// Общие сведения об этой сборке предоставляются следующим набором
```
Comment (AlexanderSoroka) [inline TheLongestSubsequence/Properties/AssemblyInfo.cs]: Do not use russian in production

File: TheLongestSubsequence/Sequence.cs [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+          }
+          counter = 1;
+        }
+      }
+      return amountOfTheSameSymbols;
+    }
+
+    protected string BuildTheOutputString()
+    {
+      int theLongestSubsequence = FindTheLongestSubsequence();
```
Comment (AlexanderSoroka) [inline TheLongestSubsequence/Sequence.cs]: single responsibility violation

**Review by AlexanderSoroka — CHANGES_REQUESTED** [review]

File: TheLongestSubsequence/Sequence.cs [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    /// <summary>
+    /// Class constructor with a parameter, which initialize class's field.
+    /// </summary>
+    /// <param name="sequence"> A sequence, which was entered in the command line. </param>
+    public Sequence(string sequence)
+    {
+      EnteredSequence = sequence;
+    }
+
+    protected int FindTheLongestSubsequence()
```
Comment (AlexanderSoroka) [inline TheLongestSubsequence/Sequence.cs]: why it's protected?

File: TheLongestSubsequence/Sequence.cs [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+      StringBuilder outputString = new StringBuilder();
+      outputString.Append("The maximum number of identical consecutive characters in the string: ").Append(theLongestSubsequence);
+      return outputString.ToString();
+    }
+
+    /// <summary>
+    /// The method, which outputs the information about the maximum length of a subsequence 
+    /// of the same consecutive symbols containing in the input string.
+    /// </summary>
+    public void GetMaxAmountOfTheSameConsecutiveSymbols()
```
Comment (AlexanderSoroka) [inline TheLongestSubsequence/Sequence.cs]: SOLID violation

**Review by AlexanderSoroka — CHANGES_REQUESTED** [review]

File: TheLongestSubsequence/InformationOutput.cs [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+    {
+      StringBuilder outputString = new StringBuilder();
+      outputString.Append("The maximum number of identical consecutive characters in the string: ").Append(theLongestSubsequence);
+      return outputString.ToString();
+    }
+
+    /// <summary>
+    /// The method, which outputs the information about the maximum length of a subsequence 
+    /// of the same consecutive symbols containing in the input string.
+    /// </summary>
```
Comment (IgorKheidorov) [inline TheLongestSubsequence/InformationOutput.cs]: Method get can't prints anything

File: TheLongestSubsequence/Sequence.cs [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+    /// <summary>
+    /// Class constructor with a parameter, which initialize class's field.
+    /// </summary>
+    /// <param name="sequence"> A sequence, which was entered in the command line. </param>
+    public Sequence(string sequence)
+    {
+      EnteredSequence = sequence;
+    }
+
+    /// <summary>
```
Comment (IgorKheidorov) [inline TheLongestSubsequence/Sequence.cs:22]: same == iidentical

File: TheLongestSubsequence/Sequence.cs [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+    public int FindTheLongestSubsequence()
+    {
+      int amountOfTheSameSymbols = 1;
+      int counter = 1;
+      for (int i = 0; i < EnteredSequence.Length - 1; i++)
+      {
+        if (EnteredSequence[i].Equals(EnteredSequence[i + 1]))
+        {
+          counter++;
+        }
```
Comment (IgorKheidorov) [inline TheLongestSubsequence/Sequence.cs:35]: How its can be happened i.Equals(EnteredSequence.Length - 2

File: TheLongestSubsequence/Sequence.cs [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+    /// <param name="sequence"> A sequence, which was entered in the command line. </param>
+    public Sequence(string sequence)
+    {
+      EnteredSequence = sequence;
+    }
+
+    /// <summary>
+    /// This method finds the longest subsequence of the same consecutive symbols.
+    /// </summary>
+    /// <returns> It returns the value, which reflects the maximum length of the subsequence. </returns>
```
Comment (IgorKheidorov) [inline TheLongestSubsequence/Sequence.cs]: Better :  maximum of identical subsequence  length

**Review by IgorKheidorov — CHANGES_REQUESTED** [review]

**Review by AlexanderSoroka — APPROVED** [review]
Over-complicated solution

**Pull request merged**